test/kitchen/drivers/hyperv-driver.yml

```diff
@@ -0,0 +1,102 @@
+provisioner:
+  name: chef_solo
+  product_name: chef
+  product_version: 13.6.4
+  install_strategy: always
+
+driver:
+  name: hyperv
+  vm_switch: <%= ENV['KITCHEN_HYPERV_SWITCH'] %>
+  dynamic_memory: true
+  <% if ENV['KITCHEN_HYPERV_MEM_MIN'] %>
+  dynamic_memory_min_bytes: <%= ENV['KITCHEN_HYPERV_MEM_MIN'] %>
+  memory_startup_bytes: <%= ENV['KITCHEN_HYPERV_MEM_MIN'] %>
+  <% else %>
+  dynamic_memory_min_bytes: 2GB
+  memory_startup_bytes: 2GB
+  <% end %>
+  <% if ENV['KITCHEN_HYPERV_MEM_MAX'] %>
+  dynamic_memory_max_bytes: <%= ENV['KITCHEN_HYPERV_MEM_MAX'] %>
+  <% else %>
+  dynamic_memory_max_bytes: 8GB
+  <% end %>
+  <% if ENV['KITCHEN_HYPERV_PROC_COUNT'] %>
+  processor_count: <%= ENV['KITCHEN_HYPERV_PROC_COUNT'] %>
+  <% end %>
+
+platforms:
+# Loop through two lists and output a total matrix of all possible platform + chef versions,
+# for both drivers
+
+
+<%
+    # TEST_PLATFORMS syntax is `short_name1,azure_full_qualified_name1|short_name2,azure_full_qualified_name1`
+    # TEST_PLATFORMS syntax is `short_name1,parent vhd folder,parent_vhd_name|...`
+    azure_test_platforms = ENV['TEST_PLATFORMS'].split('|').map { |p| p.split(',') }
```

Comment:
`azure_test_platforms` should be `hyperv_test_platforms` (or just `test_platforms` and keep it the same on all driver files).